dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/layers/MaxPooling2D.scala

```diff
@@ -0,0 +1,13 @@
+package me.shadaj.scalapy.tensorflow.api.keras.layers
+
+import me.shadaj.scalapy.tensorflow.keras.layers.{MaxPooling2D => PyMaxPooling2D}
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonUnion._
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonOption._
+import scala.language.implicitConversions
```

Comment:
Isn't `scala.language.implicitConversions` just needed/recommended if you **define** an implicit conversion and not when you use it?

Reply:
Removing it results in warning:
```
[warn]    |                                   ^^^^^^^^^^^^^^^^^^^^^^
[warn]    |Use of implicit conversion trait Function1 in package scala should be enabled
[warn]    |by adding the import clause 'import scala.language.implicitConversions'
[warn]    |or by setting the compiler option -language:implicitConversions.
[warn]    |See the Scala docs for value scala.language.implicitConversions for a discussion
[warn]    |why the feature should be explicitly enabled.
```